Integrations/integration-SymantecEndpointProtection.yml

```diff
@@ -66,8 +113,7 @@ script:
         }
         throw err;
       }
-
-    };
+    }
 
     var doAuth = function(qParams) {
       url = fixUrl(qParams.server) + 'sepm/api/v1/identity/authenticate';
```

Comment:
TBH this code was already there (by Lior) and it worked so I didn't touch it. I fixed the naming (not using "u, p, d" anymore). Regarding the authentication parameter, currently it is mandatory so it's not necessary to check, but I think it's better to leave it anyway in case we change that.